docs/public-cloud/gitlab-setup.md

```diff
@@ -0,0 +1,225 @@
+# Gitlab Setup Guide for CAF Sample App
+
+This document outlines the steps necessary to set up a sample CAF (Cloud Adoption Framework) application in GitLab. 
+
+To summarize:
+
+0. Clone the [symphony](https://github.com/aztfmod/symphon) repo.
+  - For now use the following branch: [/davesee/bump_versions](https://github.com/aztfmod/symphony/tree/davesee/bump_versions)
+1. Set up the GitLab server with ```gitlab-server-setup.sh```
+2. Set up SSH Keys and clone repostories using ```clone-repos.sh```
+3. Deploy the launchpad landing zone manually to set up MSIs, using ```rover```
+4. Set up the Gitlab runners with ```install.sh```
+5. Run pipelines in Gitlab and troubleshoot as necessary
+
+# 1. GitLab Server Ssetup
```

Comment:
mention starting the devcontainer startup

Reply:
Added "before you begin" steps to include DevContainer startup.